solutionbox/code_free_ml/mltoolbox/code_free_ml/data/trainer/feature_transforms.py

```diff
@@ -64,6 +64,8 @@
 Image.new('RGB', (16, 16)).save(_img_buf, 'jpeg')
 IMAGE_DEFAULT_STRING = base64.urlsafe_b64encode(_img_buf.getvalue())
 
+IMAGE_BOTTLENECK_TENSOR_SIZE = 2048
```

Comment:
Only the individual transform names are exposed to users. We should have documentation/help pages instead of pointing to code in our "code free solution"!

Added comments about what constants are 'safe' to change.